CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 ### Performance enhancements
 
 ### Bug fixes
+* `skbio.TreeNode.tips()` enables the `include_self` parameter if the node of interest happens to be a tip.
```

Comment:
What about: "The `include_self` was not being honored in `TreeNode.tips`. The scope of this bug was that if `tips` was called on a tip, it would always result in an empty list when unrolled."